README.md

```diff
@@ -9,6 +9,6 @@ cp genesis.json ~/.sommelier/config/genesis.json
 sommelier gentx validator {self-delegation-amount} \
     $(gorc --config $HOME/gorc/config.toml keys eth show signer) \
     $(sommelier keys show validator --bech acc -a) \
```

Comment:
Are we sure that this isn't supposed to be
```suggestion
    $(sommelier keys show orchestrator --bech acc -a) \
```

since this is getting written into the gentx under the key `orchestrator_address`

https://github.com/PeggyJV/SommelierNetwork/blob/main/gentxs/standard_crypto.json#L33

Reply:
The testnet instructions were
```
# register delegate keys for eth and orchestrator keys
sommelier tx gravity set-delegate-keys \
    $(sommelier keys show validator --bech val -a) \ # validator address
    $(gorc --config $HOME/gorc/config.toml keys cosmos show orchestrator) \ # orchestrator address (this must be run manually and address extracted)
    $(gorc --config $HOME/gorc/config.toml keys eth show signer) \ # eth signer address
    $(gorc --config $HOME/gorc/config.toml sign-delegate-keys signer $(sommelier keys show validator --bech val -a)) \ 
    --chain-id sommtest-4 \ 
    --from validator \ 
    --fees 25000usomm -y
```